api/src/test/java/uk/gov/dhsc/htbhf/claimant/testsupport/EligibilityOverrideTestDataFactory.java

```diff
@@ -4,27 +4,39 @@
 import uk.gov.dhsc.htbhf.dwp.model.EligibilityOutcome;
 
 import java.time.LocalDate;
+import java.util.List;
 
+import static java.util.Collections.emptyList;
 import static uk.gov.dhsc.htbhf.claimant.testsupport.TestConstants.OVERRIDE_UNTIL_FIVE_YEARS;
 
 public class EligibilityOverrideTestDataFactory {
 
     public static EligibilityOverride aConfirmedEligibilityOverride() {
         return aConfirmedEligibilityOverrideBuilder()
                 .overrideUntil(OVERRIDE_UNTIL_FIVE_YEARS)
+                .childrenDob(emptyList())
+                .build();
+    }
+
+    public static EligibilityOverride aConfirmedEligibilityOverrideWithChildren(List<LocalDate> childrenDob) {
+        return aConfirmedEligibilityOverrideBuilder()
+                .overrideUntil(OVERRIDE_UNTIL_FIVE_YEARS)
+                .childrenDob(childrenDob)
                 .build();
     }
 
     public static EligibilityOverride aConfirmedEligibilityWithUntilDate(LocalDate overrideUntil) {
```

Comment:
Would it be worthwhile mentioning no children in this and the next method too?

Reply:
Not for `aNotConfirmedEligibilityOverride` - there should never be children if eligibility is not confirmed.